web/src/buddy_data.ts

```diff
@@ -269,17 +273,23 @@ export function get_title_data(
         return {
             first_line: person.full_name,
             second_line: user_status.get_status_text(user_id),
-            third_line: last_seen,
+            third_line: is_deactivated
+                ? $t({defaultMessage: "This user has been deactivated."})
+                : last_seen,
             show_you: is_my_user,
+            is_deactivated,
         };
     }
```

Comment:
I don't think it makes sense to display status text for deactivated users. So I think what you probably want is an `if is_deactivated` case before this block, rather than the inline `third_line` conditional here and the `second_line` one below.